adafruit_imageload/bmp/indexed.py

```diff
@@ -82,17 +87,28 @@ def load(
 
         if compression == 0:
             chunk = bytearray(line_size)
```

Comment:
Please move the 'chunk' temporary variable down into the 'else' branch where it is used.  It's not needed when using `_bitmap_readinto`.

Reply:
I moved `chunk` to where it is used, per your recommendation.